openaddr/conform.py

```diff
@@ -1175,6 +1177,15 @@ def row_fxn_chain(sd, row, key, fxn):
 
     return row
 
+def row_fxn_first_non_empty(sd, row, key, fxn):
+    "Iterate all fields looking for first that has a non-empty value"
+    for field in fxn.get('fields', []):
+        if row[field] != None and len(row[field].strip()) > 0:
```

Comment:
This might be easier to read as:

```python
if row[field] and row[field].strip():
```

...which relies on `None` and `''` (the empty string) being falsey in Python.

Reply:
Oh, interesting!  I should've expected that in Python.  Learning new stuff every day!